video_xblock/video_xblock.py

```diff
@@ -4,18 +4,25 @@
 All you need to provide is video url, this XBlock does the rest for you.
 """
 
+import datetime
+import json
 import logging
+import os
 import pkg_resources
+import json
+import os
 
 from xblock.core import XBlock
 from xblock.fields import Scope, Boolean, Integer, Float, String
 from xblock.fragment import Fragment
 from xblock.validation import ValidationMessage
 from xblockutils.studio_editable import StudioEditableXBlockMixin
+from xmodule.fields import RelativeTime
```

Comment:
I think these fields are created especially for Xmodules, so probably they can be used in blocks.